src/index.js

```diff
@@ -178,6 +178,12 @@ class IPAddress extends Component {
             <span className="value">{details.size.toLocaleString()}</span>
             <span className="label">Count</span>
           </span>
+        </div>
+
+        <div className="break">
+        </div>
+
+        <div className="usable">
```

Comment:
Thit virtual div is to ensure the first and last usable IP's are always on their own row. I noticed working on this that the footer has the explanation about the base and broadcast. It made more sense to me to just force this to be two lines and keep the first and last near the footer footnote.